recipes/connectorx/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -ex
+
+cp $SRC_DIR/README.md $SRC_DIR/connectorx-python/README.md
+cp $SRC_DIR/LICENSE $SRC_DIR/connectorx-python/LICENSE
+
+pushd $SRC_DIR/connectorx-python
+poetry install
```

Comment:
This installs all dependencies of connectorx-python separately from PyPI. Is this really needed? Shouldn't they come from conda-forge?

Reply:
For pip, env vars are set to force no deps. Idk if this is done for poetry, and I don't know if it's possible

https://github.com/python-poetry/poetry/issues/3377

Reply:
I can try deleting this line?